diff -c MOO-1.8.1/Makefile.in ../MOO-1.8.1/Makefile.in
*** MOO-1.8.1/Makefile.in	Mon Jan 10 17:54:05 2000
--- ../MOO-1.8.1/Makefile.in	Sat May 20 00:53:37 2000
***************
*** 13,23 ****
  #   Palo Alto, CA 94304
  #   Pavel@Xerox.Com
  
! CC = @CC@
! LIBRARIES = @LIBS@
  YACC = @YACC@
  
! CFLAGS = -O
  # If you're using GCC, you may prefer:
  # CFLAGS = -O2 -finline-functions
  #
--- 13,31 ----
  #   Palo Alto, CA 94304
  #   Pavel@Xerox.Com
  
! # 
! # with autoconf2 I'll go edit it to support --with-expat=
! # Until then, set this to where you unpacked and compiled expat
! #
! EXPAT = /home/fox/src/dist/expat
! 
! EXPAT_OBJS = $(EXPAT)/xmltok/*.o $(EXPAT)/xmlparse/*.o 
! 
! CC = @CC@ -I$(EXPAT)/xmlparse
! LIBRARIES = @LIBS@ $(EXPAT_OBJS)
  YACC = @YACC@
  
! CFLAGS = -O -DEXPAT_XML
  # If you're using GCC, you may prefer:
  # CFLAGS = -O2 -finline-functions
  #
***************
*** 35,41 ****
  	log.c malloc.c match.c md5.c name_lookup.c network.c net_mplex.c \
  	net_proto.c numbers.c objects.c parse_cmd.c pattern.c program.c \
  	property.c quota.c ref_count.c regexpr.c server.c storage.c streams.c str_intern.c \
! 	sym_table.c tasks.c timers.c unparse.c utils.c verbs.c version.c
  
  OPT_NET_SRCS = net_single.c net_multi.c \
  	net_mp_selct.c net_mp_poll.c net_mp_fake.c \
--- 43,49 ----
  	log.c malloc.c match.c md5.c name_lookup.c network.c net_mplex.c \
  	net_proto.c numbers.c objects.c parse_cmd.c pattern.c program.c \
  	property.c quota.c ref_count.c regexpr.c server.c storage.c streams.c str_intern.c \
! 	sym_table.c tasks.c timers.c unparse.c utils.c verbs.c version.c ext-xml.c
  
  OPT_NET_SRCS = net_single.c net_multi.c \
  	net_mp_selct.c net_mp_poll.c net_mp_fake.c \
diff -c MOO-1.8.1/extensions.c ../MOO-1.8.1/extensions.c
*** MOO-1.8.1/extensions.c	Mon Dec 14 05:17:52 1998
--- ../MOO-1.8.1/extensions.c	Sat May 20 00:54:47 2000
***************
*** 172,177 ****
--- 172,180 ----
  }
  #endif
  
+ #ifdef EXPAT_XML
+ extern void register_xml(void);
+ #endif
  
  void
  register_extensions()
***************
*** 183,188 ****
--- 186,194 ----
  #ifdef STUPID_VERB_CACHE
      register_function("log_cache_stats", 0, 0, bf_log_cache_stats);
      register_function("verb_cache_stats", 0, 0, bf_verb_cache_stats);
+ #endif
+ #ifdef EXPAT_XML
+     register_xml();
  #endif
  }
  
Common subdirectories: MOO-1.8.1/pgperf and ../MOO-1.8.1/pgperf
diff -c MOO-1.8.1/storage.h ../MOO-1.8.1/storage.h
*** MOO-1.8.1/storage.h	Mon Dec 14 05:19:00 1998
--- ../MOO-1.8.1/storage.h	Sat May 20 00:54:59 2000
***************
*** 34,39 ****
--- 34,40 ----
  
      M_REF_ENTRY, M_REF_TABLE, M_VC_ENTRY, M_VC_TABLE, M_STRING_PTRS,
      M_INTERN_POINTER, M_INTERN_ENTRY, M_INTERN_HUNK,
+     M_XML_DATA,
  
      Sizeof_Memory_Type
  
